freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/keytab/KeerberosMgmtV1Endpoint.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.freeipa.api.v1.keytab;
+
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.sequenceiq.freeipa.api.v1.keytab.doc.KeytabModelNotes;
+import com.sequenceiq.freeipa.api.v1.keytab.model.ServiceKeytabRequest;
+import com.sequenceiq.freeipa.api.v1.keytab.model.ServiceKeytabResponse;
+import com.sequenceiq.freeipa.api.v1.keytab.model.UserKeytabRequest;
+import com.sequenceiq.freeipa.api.v1.keytab.model.UserKeytabResponse;
+import com.sequenceiq.freeipa.api.v1.keytab.doc.KeytabOperationsDescription;
+import com.sequenceiq.service.api.doc.ContentType;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Path("/v1/kerberosmgmt")
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(value = "/v1/kerberosmgmt", protocols = "http,https")
+public interface KeerberosMgmtV1Endpoint {
```

Comment:
there is a plus 'e'
regarding naming: everything is keytab related, maybe it should be keytab mgmt endpoint if we don't plan to have any extra functionality not related to keytabs

don't forget to add this to "FreeIpaApiEndpoint"

Reply:
There will be more APi's added in the coming days to get the TGT as well. That is why i picked kerberosmgmt rather than keytabmgmt

Reply:
I agree that the naming is a little funny since you've placed it in the api.v1.keytab package

Reply:
I updated the package name accordingly.

Reply:
It is addressed in https://github.com/hortonworks/cloudbreak/pull/5441